dts/bindings/flash_controller/nxp,iap-msf1.yaml

```diff
@@ -3,6 +3,6 @@
 
 description: NXP MSF1 Flash Memory Module (FMU)
 
-compatible: "nxp,iap-msf1"
+compatible: "nxp,msf1"
```

Comment:
Should you also rename the file name to nxp,msf1.yaml?

Reply:
renamed